src/pykeen/nn/node_piece/representations.py

```diff
@@ -166,6 +167,22 @@ def _plain_forward(
         return self.vocabulary(token_ids)
 
 
+class DiversityInfo(NamedTuple):
```

Comment:
```suggestion
class HashDiversityInfo(NamedTuple):
```
This would be a bit more specific, and only slightly longer.

Reply:
done in 1c69953

Reply:
Thanks!